src/Lykke.Service.PayInternal.Services/CalculationService.cs

```diff
@@ -118,8 +116,11 @@ public class CalculationService : ICalculationService
                 }
             }
 
-            await _log.WriteInfoAsync(nameof(CalculationService), nameof(GetRateAsync),
-                new {askPrice, bidPrice}.ToJson(), "Market rate that will be used for calculation");
+            _log.Info("Market rate that will be used for calculation", new
```

Comment:
Alexey Novicikhin usually suggests to save in context only data wich is used for searching by fields in kibana.